package.json

```diff
@@ -9,7 +9,7 @@
     "url": "https://github.com/cardsforhumanity/cardsforhumanity.git"
   },
   "engines": {
-    "node": "8.9.1",
+    "node": "8.9.0",
```

Comment:
which version was agreed upon? `8.9.0` or `8.9.1`?

Reply:
changed it to 8.9.0